application/classes/Ushahidi/Repository/User.php

```diff
@@ -79,7 +79,7 @@ public function update(Entity $entity)
 	// SearchRepository
 	public function getSearchFields()
 	{
-		return ['email', 'role', 'q' /* LIKE realname, email */];
+		return ['email', 'roles', 'q' /* LIKE realname, email */];
```

Comment:
I don't think the param name should change. Our param names are still singular even when they allow multiple values.